src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesCloud.java

```diff
@@ -107,6 +110,7 @@
     private boolean capOnlyOnAlivePods;
 
     private String namespace;
+    private boolean directConnection;
```

Comment:
To enable it by default for new instances (and also for unit tests)

```suggestion
    private boolean directConnection = true;
```

Reply:
Done. I also had to adjust/fix the tests.

Reply:
Missed the tests running only in K8s clusters.
Is `Jenkinsfile-kubernetes` the easiest way to test this "locally"?

Reply:
yes, that's the one